src/components/Controls.svelte

```diff
@@ -69,18 +69,23 @@
 
 <div class="controls">
 	<div class="controlButtons">
-		<button class="playPause" on:click|stopPropagation={toggleAudio}>
-			<span class="material-icons">
-				{currStatus === status.PLAY ? 'pause' : 'play_arrow'}
-			</span>
-		</button>
-		<input
-			type="range"
-			max="30"
-			value={position}
-			class="trackPlaceholder"
-			on:click|stopPropagation|self={seek}
-		/>
+		{#if audio.src != ''}
```

Comment:
!== vs !=
educate me on what the difference is and if line 72 should be updated

Reply:
Mistake on my part. !== checks type while != only checks value.